python/lsst/meas/deblender/baseline.py

```diff
@@ -1119,7 +1310,53 @@ def _overlap(xlo, xhi, xmin, xmax):
 
 
 def _handle_flux_at_edge(log, psffwhm, t1, tfoot, fp, maskedImage,
-                         x0, x1, y0, y1, psf, pk, sigma1, patchEdges):
+                         x0, x1, y0, y1, psf, pk, sigma1, patchEdges
+    ):
+    """
+    If a substantial amount of flux is found at the edge of the template,
```

Comment:
Actually, on second look: checking for "substantialness" ("substance"? "substantiality"?) isn't performed by this function. From a very quick reading (do correct me if I'm wrong), it's actually `deblend` which decides if there is significant flux at the edge, and hence whether to call this. Presumably this still runs if it's called with a template which doesn't *actually* have substantial flux at the edge, so the docstring shouldn't imply it's required.